scripts/build/Platform/Windows/pipeline.json

```diff
@@ -16,5 +16,30 @@
         "nightly-clean": {
             "CLEAN_WORKSPACE": true
         }
+    },
+    "PIPELINE_JENKINS_PARAMETERS": {
```

Comment:
I think the batch script supports another parameter called COMMIT_ID, which is used to lock the version of CDK applications to deploy.

Reply:
I will add COMMIT_ID as a new parameter too, thanks!